app/services/submissions/search_service.rb

```diff
@@ -28,6 +28,7 @@ def search_query
       relation = relation.where("has_been_assigned_to ? :caseworker_id", caseworker_id:) if caseworker_id
       relation = relation.where(assigned_user_id:) if assigned_user_id
       relation = relation.where(account_number:) if account_number
+      relation = relation.where(high_value:) if high_value
```

Comment:
This works for the current requirement, but it's misleading. As it stands it makes it impossible to filter by NOT high value, because if `search_params[:high_value]` is set to false, then `if high_value` will prevent this line from running. You'd be better off with `unless high_value.nil?`